formwidgets/mltext/partials/_mltext.htm

```diff
@@ -17,7 +17,7 @@
             placeholder="<?= e(trans($field->placeholder)) ?>"
             class="form-control"
             autocomplete="off"
-            maxlength="255"
+            <?= $field->hasAttribute('maxlength') ? '' : 'maxlength="255"' ?>
```

Comment:
This change is not related to the current PR, can you create an independent PR if you want to fix this?